dist/tools/flatc/Makefile

```diff
@@ -1,14 +1,14 @@
-PKG_NAME=flatc
+PKG_NAME=flatbuffers
 PKG_URL=https://github.com/google/flatbuffers
 PKG_VERSION=9e7e8cbe9f675123dd41b7c62868acad39188cae
 PKG_LICENSE=Apache2.0
 
-PKG_BUILDDIR=$(CURDIR)/bin
-
 # manually set some RIOT env vars, so this Makefile can be called stand-alone
 RIOTBASE ?= $(CURDIR)/../../..
 RIOTTOOLS ?= $(CURDIR)/..
 GITCACHE ?= $(RIOTTOOLS)/git/git-cache
+PKGDIRBASE ?= $(RIOTBASE)/packages
+PKG_BUILD_DIR ?= $(CURDIR)/build
```

Comment:
why build here and bin above?

Reply:
I think the bin above is a left-over, I'll check.

Reply:
There's no bin for flatc. I kept for the other tools (bossa, mosquitto, edbg) just to keep the previous location (otherwise gitignore would also need to be updated, not difficult but will do more changes)

Reply:
Ok, for consistency with other tools, now there's no more `build` directory, only `bin`s